src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,221 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.exception.MissingAtEventException;
+import duke.exception.MissingByDeadlineException;
+import duke.exception.MissingDetailsException;
+import duke.exception.UnknownDateTimeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Command {
+    //Custom byeCommand Method to exit Duke
+    public static String byeCommand() {
+        return ("BYE");
+    }
+
+    /**
+     * calendarCommand Method finds Tasks that finds, matches and displays Tasks
+     * on a specific date.
+     *
+     * @param taskList      is the list of Tasks are saved and manipulated
+     * @param commandSuffix is the additional String that accompanies two-step commands
+     * @throws UnknownDateTimeException when improperly formatted DateTime values are given
+     */
+    public static String calendarCommand(TaskList taskList, String commandSuffix) throws UnknownDateTimeException {
+        try {
+            DateTimeFormatter dtFormat = DateTimeFormatter.ofPattern("d/M/yyyy");
+            LocalDate calendarDate = LocalDate.parse(commandSuffix, dtFormat);
+            return calendarFind(taskList, calendarDate);
+        } catch (IndexOutOfBoundsException | DateTimeParseException e) {
+            throw new UnknownDateTimeException();
+        }
+    }
+
+    //Custom calendarFind Method to find and print the list with the horizontal borders + running index
+    static String calendarFind(TaskList taskList, LocalDate calendarDate) throws DateTimeParseException {
+        StringBuilder sb = new StringBuilder();
+        List<String> calendarList = new ArrayList<>();
```

Comment:
Plural form should be used on names representing a collection of objects. So maybe you can name calendarList as dates.

Reply:
I agree, but in this case because I am already using the noun **_"List"_** the plurality is already there with better concreteness. However, I did contemplate naming this `dateList` but figured that might actually be more confusing than keeping it in-line with the method's name - for example, does `dateList` refer to the list of dates a user has passed in, or the list of "dated tasks" to be returned?

_For reference: https://softwareengineering.stackexchange.com/questions/352359/naming-a-class-representing-a-collection-of-objects-of-another-class_